test/forge/utils/DeployRouter.sol

```diff
@@ -28,20 +28,22 @@ contract DeployRouter is Test {
     address public constant ROUTER_REWARDS_DISTRIBUTOR = 0x0000000000000000000000000000000000000000;
     address public constant LOOKSRARE_REWARDS_DISTRIBUTOR = 0x0554f068365eD43dcC98dcd7Fd7A8208a5638C72;
 
-    function deployRouterMainnetConfig() public returns (UniversalRouter router) {
-        return deployRouter(address(0));
-    }
+    address internal constant RECIPIENT = 0xaAaAaAaaAaAaAaaAaAAAAAAAAaaaAaAaAaaAaaAa;
+    address internal constant MAINNET_PERMIT2 = 0x000000000022D473030F116dDEE9F6B43aC78BA3;
 
-    function deployFixtureMainnetConfig() public returns (UniversalRouter router, Permit2 permit2) {
-        try vm.envBool("USE_MAINNET_DEPLOYMENT") returns (bool useMainnet) {
-            if (useMainnet) return useMainnetDeployment();
-        } catch {}
-        permit2 = new Permit2();
-        router = deployRouter(address(permit2));
-    }
+    address internal constant UNIVERSAL_ROUTER_V1 = 0xEf1c6E67703c7BD7107eed8303Fbe6EC2554BF6B; // available from block 16075466
```

Comment:
why do we need these around still?

Reply:
yep good spot - was here as a remnant of an issue with 1 NFT protocol